### ISSUE TITLE
In the explanatory example, "supports" should be "supportsRequirement"?

### ISSUE THREAD

**Issue by jimjyang** [SEMICeu/CCCEV#49]
[The explanatory example](https://semiceu.github.io/CCCEV/releases/2.00/html/CCCEV_AnExplanatoryExample.pdf) linked to from [the Summary section](https://semiceu.github.io/CCCEV/releases/2.00/#summary) is very useful! 

However, the example uses the property name/URI `supports` from `:Evidence` to `:InformationRequirement` and `:Constraint`, and it should be `supportsRequirement` according to CCCEV 2.00.

**Comment by bertvannuffelen** [comment]
Good point, the example has been made on the basis of an earlier draft.
